Yoast/wordpress-seo
# Bump plugin grunt tasks

Make sure `build` keeps working.

This PR can be summarized in the following changelog entry:

* None needed.

## Test instructions

This PR can be tested by following these steps:

* Use `grunt:build`, make sure it still works.

## Documentation

## Quality assurance


Fixes #
